drivers/input/Kconfig.npcx

```diff
@@ -1,43 +1,43 @@
-# NPCX Keyboard Scan driver configuration options
+# NPCX Keyboard scan driver configuration options
 
 # Copyright (c) 2022 Nuvoton Technology Corporation.
 # SPDX-License-Identifier: Apache-2.0
 
-menuconfig KSCAN_NPCX
-	bool "Nuvoton NPCX embedded controller (EC) Keyboard Scan (KSCAN) driver"
+menuconfig INPUT_NPCX_KBD
```

Comment:
This probably counts as an API change, or at least something that should be noted in the release notes.

Reply:
Oh yeah good point, I already have a list ready there, added a commit to add a line about this. Kept the Kconfig name generic since the next step here is to pull them out of that driver into generic code anyway, will document that once I've figured how (likely change them to devicetree options).